hack/e2e-test.sh

```diff
@@ -41,7 +42,9 @@ function cleanup {
 function startup {
     if [ $CREATE_KIND_CLUSTER == 'true' ]
     then
-        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION --config $SOURCE_DIR/kind-cluster.yaml
+        $KIND create cluster --name $KIND_CLUSTER_NAME --image $E2E_KIND_VERSION --config $SOURCE_DIR/kind-cluster.yaml --wait 1m
```

Comment:
what happens if the cluster is not ready? do the commands below run?

Reply:
Yes, the command below runs since kind finishes with exit 0 even if the cluster isn't ready. 

```
kind create cluster --wait 1s
Creating cluster "kind" ...
 ✓ Ensuring node image (kindest/node:v1.27.1) 🖼 
 ✓ Preparing nodes 📦  
 ✓ Writing configuration 📜 
 ✓ Starting control-plane 🕹️ 
 ✓ Installing CNI 🔌 
 ✓ Installing StorageClass 💾 
 ✗ Waiting ≤ 1s for control-plane = Ready ⏳ 
 • WARNING: Timed out waiting for Ready ⚠️
Set kubectl context to "kind-kind"
You can now use your cluster with:

kubectl cluster-info --context kind-kind

Have a nice day! 👋
```

Reply:
Awesome
/lgtm